### ISSUE TITLE
Option `es-nolists` doesn't disable the use of `ñ` as a counter

### ISSUE THREAD

**Issue by benjavalero** [jbezos/babel-spanish#9]
I guess this has been discussed in the past, so sorry for repeating in that case.

I think the option `es-nolists` shoud disable the use of the Spanish letter `ñ` as a counter, as it is a something related to lists.

I know there exists the option `es-nolayout` to do that, but it has a wider impact than `es-nolists`.

**Comment by jbezos** [comment]
No, no se ha tratado antes. En realidad `\alph` puede aparecer en más contextos, como llamadas de nota. Tal vez una opción adicional `no-alph` sea una alternativa (aunque no quería añadir más opciones).
